docs: containerization (#268)
logto-io/docs: docs/docs/recipes/deployment/README.md

### Database alteration

> If a schema change is inevitable, we will provide an alteration script. Simply run "pnpm alteration deploy" with ease to upgrade your database schema.

Reviewer: ```suggestion
If a schema change is inevitable, we will provide an alteration script. Simply run `pnpm alteration deploy` with ease to upgrade your database schema.
```

> If a schema change is inevitable, we will provide an alteration script. Simply run "pnpm alteration deploy" with ease to upgrade your database schema.

Reviewer: ```suggestion
If a schema change is inevitable, we will provide an alteration script. Simply run `npm alteration deploy` in the Logto project root with ease to upgrade your database schema.
```

## Containerization

For production, you may use Docker to containerize Logto. You can find the Dockerfile in the root directory of the project. If you want to run mutiple instances of Logto, for instance, deploy Logto in a Kubernetes cluster, There are some additional steps you need to take.

> ### Share connectors folder

Reviewer: ```suggestion
### Shared connectors folder
```
